apis/network/v1beta1/common_webhook.go

```diff
@@ -0,0 +1,78 @@
+package v1beta1
+
+// Because webhooks *MUST* exist within the api/<version> package, we need to place common
+// functions here that might be used across different Kinds' webhooks
+
+import (
+	"context"
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	goClient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Client needed for API calls (manager's client, set by first SetupWebhookWithManager() call
+// to any particular webhook)
+var webhookClient goClient.Client
```

Comment:
It looks like only `ipset_webhook.go` ever sets the `webhookClient` right now.  Just wondering if you intended to add it to all webhooks given what the comment says.

Reply:
probably not to all, but I'll add webhook to dns which will also required it. thats why I added it here